### PR TITLE
DCOS_OSS-653: Handle {message: null} marathon response

### PR DESCRIPTION
**Pull request by wavesoft**
This PR ensures that the `MarathonErrorUtil` utility properly parses `{message: null}` marathon error responses.

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [x] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by mlunoe** [comment]
Thanks for doing this! @leemunroe and @sascala can we get a sign-off on the wording?
This error will show when interacting with Marathon and it returns an empty error message. In most cases, it will provide information, but we are still discovering issues where this emerges.
I have opened an issue where I discovered this error message to trigger: https://github.com/mesosphere/marathon/issues/5252

**Comment by sascala** [comment]
@mlunoe Apart from putting a period at the end of the sentence, the wording LGTM.

**Comment by nLight** [comment]
@wavesoft Flicts!

File: plugins/services/src/js/utils/MarathonErrorUtil.js [hunk 1]
```diff
@@ -51,6 +51,17 @@ const MarathonErrorUtil = {
 
     // Both `details` and `errors` can be missing
```
Comment (nLight) [inline plugins/services/src/js/utils/MarathonErrorUtil.js]: Both `details` and `message`?

**Comment by nLight** [comment]
Please resolve conflicts and let's update that comment before merging 💪

**Comment by nLight** [comment]
run unit tests

**Comment by nLight** [comment]
run unit tests

**Pull request merged**